Lib/fontTools/cu2qu/__main__.py

```diff
@@ -1,6 +1,7 @@
 import sys
 from .cli import main
 
+description = "Convert a UFO font with cubic curves to quadratic curves"
```

Comment:
Shouldn’t it be `__doc__` or a similarly named variable?

Reply:
Yeah, using docstrings is clearly the right approach. OK, rewrite incoming...